### PR TITLE
MC7000 doc - Pad mode with pictures

### PR DESCRIPTION
**Pull request by toszlanyi**
Hey there - I have changed the documentation file to use pictures instead of text based visualization for PAD buttons. Also re-arranged the different blocks.

### CODE REVIEW

**Comment by Holzhaus** [comment]
Thanks. I noticed that the SVG files contain raster images (png). Is that intended? Could you replace them with actual vector graphics?

**Comment by toszlanyi** [comment]
> Is that intended?

No it's not. Have copied them from the manual pdf file into Libre Office Impress and then exported from there to svg. If you could suggest any other software on Ubuntu that is capable writing svg then let me know. BTW. How you recognize that png is inside svg? I would like to check before I send updates then. Thanks a lot.

**Comment by Holzhaus** [comment]
How you recognize that png is inside svg?

Check the SVG source code, it contains a base64gencoded png. Grep for `png` to verify. I noticed it when zooming in and seeing the individual pixels.

File: source/_static/controllers/denon_mc7000_beatjump_mode.svg [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
...6W9LbUJJbLFYdfTCWK5JpBtZbqQ2i7qjd29UypQ3yZouhmlz9brJjqGWxMtqIj8aqS4lV/c12/qyDSGPDhcmUBq1Hw17+RvOYryQLeLmhk+tbcKJlGbsFKA2m12uS9Vda0a6oVOTPkuGnOE3ANOPf2slCorOS+SpphvW0j+W6OJ0X9XrbdvagJTXdLpM2xpujSBM9rUNavPvrsZyqjeG/06j2e5b8aTB70PYk0cciSXvcbadLk6ppkR/pJvuChN6sbg/ZWWOnO35XWL3WuvXPGqsu0Ye3hPkYmy2GaZ26vptlwsLkYtVurVu31GL5C0uy4khuSnq2vFzW7Z5JLcP1kfWseUus5Of6+6laplcm85VdktATqwZBMpdBtkVe/tRi1WTaQ2qJ17ZOdxYtajK8NsyfKnltmEMDWKvKVfrYjfrUn+jPBF3y/c+GDfVmQpPkszSz/0LSk6WiWF5d68HemNUZ3ct4U70md1kDqPXrHTY0Pe3lDVrxR+iS73/W+6A3Y5JOIn2pNkafFUuyNHBp4vy+XpTJAma57F7n0h6H7dYbYcPR+tT/nT7N0WL9LsSqqsaqM0p1udF6spI2MparehEXLFZpmS6x5F2sZ8vJpVq/PlzzcimxajDkY5XWpfN1W/5Qt906m0ITF9DRsKhEKyRSb9S9xbfa3mLSMIoc3TMkm+9/2WHRdduSaWZvXd6X1XoYJOyACv0CnK3VZ6slfjb/r5Q+FWud4qxtfa7xUPusMbQds17ufNVq/l0aq2y+1tZsODp7jljV/2bEKkk6uHaWy/KmXh6AZcmUxqoXJvPERDblYhV1agDkY5V1mNoW0xbmZq1jm4uIVcd1ZnjbW9TKdltyeGtJXqclfp0+vLRDZUPRnNyte2Lfllh+0C+Tg8PchjfafB8fyat07/WXtZwX9bpcWACbt1mXb9ZFvtcve/dNYtVr9OHlvS4X2mOnFNJE+R+kJv+k9erbmmT1Q6yy3tB3aay6JcSql2tBXyki7BQOCE2ybtX2727dGfyWrxcXq477Ejyhsn1ZOLwVHRTm89jPtb0uC04MxCoUIRergKa41iEAAEBBSLQAAAAKQqIFAABQEBItAACAgpBoAQAAFIRECwAAoCAkWgAAAAUh0QIAACgIiRYAAEBBSLQAAAAKQqIFAABQkP8fcdMgmgNRDSQAAAAASUVORK5CYII="/>
```
Comment (Holzhaus) [inline source/_static/controllers/denon_mc7000_beatjump_mode.svg:131]: Here is the png

**Comment by ronso0** [comment]
> If you could suggest any other software on Ubuntu that is capable writing svg then let me know.

Inkscape is great for that.

**Comment by Holzhaus** [comment]
@toszlanyi have you tried to open the manual PDF directly with Inkscape? If the PDF contains the vector image, you'll be able to edit it without rasterization.

**Comment by toszlanyi** [comment]
> @toszlanyi have you tried to open the manual PDF directly with Inkscape? If the PDF contains the vector image, you'll be able to edit it without rasterization.

No - I didn't before. And I find Inkscape quite confusing. But this got me to try and import the pdf directly in LibreOffice Draw which was perfect and all graphics as vector - just copied in a new Draw document and got my own text in there, exported as svg and done. Have also just redone the overview image which is half the size now. Hope all is good so far...

**Review by Holzhaus — COMMENTED** [review]
Nice, the SVGs are now true vector images. Unfortunately the text is not rendered correctly, probably because I don't have that font installed. Can you convert the text to paths?

**Comment by toszlanyi** [comment]
>Can you convert the text to paths?
All text is converted to lines. The deploy-preview looks good on my mobile phone too.

**Comment by toszlanyi** [comment]
What do you think about one little change to the overview graphics by substituting "Serato" with "Mixxx" like in that picture? Its just a little thing but I don't like to have Serato shown on there - got a Mixxx sticker glued over it on my hardware too ;) If you agree I commit that little change as well.

grafik

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you. It's really great how good the controller docs have become since we started putting them into the manual.

> What do you think about one little change to the overview graphics by substituting "Serato" with "Mixxx" like in that picture?

Although I personally find it amusing, we should probably try match the controller as close as possible to not confuse new users.

That said, I'm kind of jealous that I don't have any mixxx stickers...

**Comment by toszlanyi** [comment]
> That said, I'm kind of jealous that I don't have any mixxx stickers... 

Haha - it's just a matter to put the logo jpg onto a stick and walk into your next DM store. There you can print stickers.

**Pull request merged**